src/bindings.c

```diff
@@ -611,33 +619,32 @@ server_rec *pr_ipbind_get_server(const pr_netaddr_t *addr, unsigned int port) {
       "'%s' listening on wildcard address", pr_netaddr_get_ipstr(addr), port,
       ipbind->ib_server->ServerName);
     return ipbind->ib_server;
+  }
 
-  } else {
```

Comment:
Style cleanup here; no functional change.  I'm becoming a fan of early returns, and removing unnecessary `else` blocks where possible.